contributors/guide/bug-bounty.md

```diff
@@ -0,0 +1,67 @@
+# Kubernetes Bug Bounty Program
+
+This program is a **work in progress**. This tracks the currently proposed program, but the bug
+bounty is not currently active.
```

Comment:
Add a note: if you currently have a bug to submit, follow instructions at https://kubernetes.io/docs/reference/issues-security/security/

Reply:
Done.